WooCommerce/WooCommerceTests/ViewRelated/Dashboard/StoreStatsV4ChartAxisHelperTests.swift

```diff
@@ -0,0 +1,38 @@
+import XCTest
+@testable import WooCommerce
+
+class StoreStatsV4ChartAxisHelperTests: XCTestCase {
+    private let helper = StoreStatsV4ChartAxisHelper()
+
+    func testDatesAcrossTwoMonths() {
+        // GMT: Saturday, June 1, 2019 12:29:29 AM
+        let dateInJune = Date(timeIntervalSince1970: 1559348969)
+        // GMT: Monday, June 10, 2019 12:29:29 AM
+        let secondDateInJune = Date(timeIntervalSince1970: 1560126569)
+        // GMT: Thursday, August 1, 2019 12:29:29 AM
+        let dateInAugust = Date(timeIntervalSince1970: 1561940969)
```

Comment:
Did you mean to test August 1st or July 1st? 

>Monday, July 1, 2019 12:29:29 AM

Both are consistent July 1st dates based on Epoch time, so the unit tests pass.

Reply:
ohh great catch, I'll update the timestamp!